### ISSUE TITLE
code error

### ISSUE THREAD

**Issue by Jamesli0123** [LIX-shape-analysis/SURFMNet#6]
Hi, thanks for your great work. when I run the[ train_DFMnet.py](https://github.com/LIX-shape-analysis/SURFMNet/blob/master/train_DFMnet.py) ，the error appears which will show in the follow and I cannot figure out.
InternalError (see above for traceback): tensorflow/core/kernels/cuda_solvers.cc:375: cuSolverDN call failed with status =7
	 [[Node: matrix_solve_ls_1/Cholesky = Cholesky[T=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:GPU:0"](matrix_solve_ls_1/MatMul)]]
	 [[Node: Adam/update/_1642 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device_incarnation=1, tensor_name="edge_8234_Adam/update", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:CPU:0"]()]]
looking forward for your reply.

**Comment by bach-zouk** [comment]
Hi,

Can you please specify or check the learning rate and point cloud samples?

**Comment by cathylacus** [comment]
Hi，How you resolve this problem? I also find this error.

**Comment by Jamesli0123** [comment]
use CUDA-Toolkit 9.0 Cudnn7.6.5 TensorFlow -GPU1.13
